docs/design/proposals/encryption-with-vault-tokens.md

```diff
@@ -245,8 +245,6 @@ metadata:
 stringData:
   cert: |
     MIIC2DCCAcCgAwIBAgIBATANBgkqh...
```

Comment:
This doesn't seems valid to me as we cannot have both certs in single secret. I think we need to remove this section
@nixpanic wdyt?

Reply:
Probably the phrasing should be changed if it is confusing? The idea is to have the key and certificate in the same Secret. Maybe rename the combined Secret from `vault-certificates` to `vault-client-auth`. 

The phrase above should then be modified similar from

> a single secret for the certificates and update the configuration to fetch certificates from a secret and update the configuration to fetch certificates

to 

> a single Secret that contains both the client authentication and update the configuration to fetch the certificate and key from the Secret

Reply:
sound good!. we can have only one cert in the secret not more than that